nncf/quantization/fake_quantize.py

```diff
@@ -230,10 +231,12 @@ def calculate_quantizer_parameters(
     else:
         num_bits = quantizer_config.num_bits
         if quantizer_config.mode == QuantizationMode.SYMMETRIC:
-            _, _, levels = calculate_symmetric_level_ranges(num_bits, signed=True, narrow_range=narrow_range)
+            level_low, level_high = calculate_symmetric_level_ranges(num_bits, signed=True, narrow_range=narrow_range)
+            levels = get_num_levels(level_low, level_high)
             input_low, input_high = symmetric_range(min_values, max_values, levels, quantizer_config, quant_group)
         else:
-            _, _, levels = calculate_asymmetric_level_ranges(num_bits, narrow_range=narrow_range)
+            level_low, level_high = calculate_asymmetric_level_ranges(num_bits, narrow_range=narrow_range)
```

Comment:
What do you think about add function like `calculate_asymmetric_num_level(num_bits, narrow_range=narrow_range)`. Looks like unnecessary code to get `num_levels`